presto-main/src/main/java/com/facebook/presto/sql/planner/sanity/NoApplyNodeLeftChecker.java

```diff
@@ -39,7 +39,7 @@ public Object visitApply(ApplyNode node, Object context)
                     throw new IllegalArgumentException("Not supported subquery");
                 }
                 else {
-                    throw new IllegalArgumentException("Correlated subquery is not yet supported");
+                    throw new IllegalArgumentException("Not supported correlated subquery");
```

Comment:
Why? The new phrasing is weird, too. I'd probably say "Unsupported correlated subquery type"